views/includes/page-templates/managing-officer-corporate.html

```diff
@@ -75,8 +75,12 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
         name: "contact_email",
         value: contact_email
       }) }}
-
-      {% include "includes/inset-text.html" %}
+      
+      {{ govukInsetText({
+        html: '<h2 class="govuk-heading-m">What information we’ll show on the public register</h2>
```

Comment:
Kept is as 
"What information we’ll show on the public register" 
not
"What information we'll shown on the public register"
as shown in the prototype.